docs/porting/custom-target-porting.md

```diff
@@ -0,0 +1,163 @@
+# Porting Mbed OS to a custom board
+
+If you are designing a board similar to an existing Mbed Enabled board, you can use target inheritance to configure your board to meet your needs. In previous versions of Mbed OS (before Mbed OS 5.8), adding a new target required modifying the `mbed-os` directory tree itself, but now you can extend an existing MCU port with a `custom_target.json` file and by adding source files that complete a port outside of `mbed-os`.
+
+For detailed information about all the ways you can configure targets, go to [adding and configuring targets](../reference/adding-and-configuring-targets.html).
+
+## Extending an existing Mbed Enabled MCU target configuration
```

Comment:
Isn't Mbed Enabled a specific thing, and not just any board that happens to already have a port?